src/strategies/catchup.jl

```diff
@@ -62,27 +62,3 @@ function update!(strat::FixedFractionalCatchup, sim)
     strat.minval = minimum(sim.logdos[sim.logdos .> 0])
```

Comment:
```suggestion
    strat.minval = minimum(sim.logdos[sim.logdos .> 0]; init = 0.0)
```

I realised there is a potential bug here if an array of zeros is passed in